src/main/java/io/vertx/ext/jdbc/impl/actions/JDBCSQLRowStream.java

```diff
@@ -43,6 +48,7 @@
   private final AtomicBoolean stClosed = new AtomicBoolean(false);
   private final AtomicBoolean rsClosed = new AtomicBoolean(false);
   private final AtomicBoolean more = new AtomicBoolean(false);
+  private final Deque<JsonArray> accumulator = new ArrayDeque<>(ACCUMULATOR_SIZE);
```

Comment:
I'm not sure about this. This value is very specific and I believe in 99.9% cases it will be by default.
My investigations shows me, that decreasing `ACCUMULATOR_SIZE` (say from 128 to 10) causes performance degradation, but increasing changes nothing.

But it is not principal for me.